rfqmsg/buy_accept.go

```diff
@@ -47,6 +50,7 @@ func NewBuyAcceptFromRequest(request BuyRequest,
 		Version:   latestBuyAcceptVersion,
 		ID:        request.ID,
 		AssetRate: assetRate,
+		AgreedAt:  time.Now(),
```

Comment:
could add a doc that describes how the `AgreedAt` is set within `NewBuyAcceptFromRequest`.

i.e if any other call-site attempts to recreate things from the request format they'll silently set a new `AgreetAt` value.

We don't have such a call site and don't think we'll do ever, but a doc would still be nice